fenix/app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -217,6 +218,33 @@ private fun PlaceholderBookmarkImage() {
     )
 }
 
+@Composable
+private fun FallbackBookmarkImage(
+    bookmark: RecentBookmark,
+) {
+    if (!bookmark.url.isNullOrEmpty()) {
+        components.core.icons.Loader(bookmark.url) {
+            WithIcon { icon ->
+                Box(
+                    modifier = imageModifier.background(
+                        color = FirefoxTheme.colors.layer2,
+                    ),
+                    contentAlignment = Alignment.Center,
+                ) {
+                    Image(
+                        painter = icon.painter,
+                        contentDescription = null,
+                        modifier = Modifier
+                            .size(36.dp)
+                            .clip(cardShape),
+                        contentScale = ContentScale.Crop,
+                    )
+                }
+            }
+        }
+    }
```

Comment:
Looks like we are creating the same component as we have on https://github.com/mozilla-mobile/firefox-android/blob/de484f54e38ecadb96bacbcd463d72d79a63d55b/fenix/app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt#L185-L201, should we reuse the component instead?